app/models/Movie.scala

```diff
@@ -0,0 +1,22 @@
+package models
+
+import java.time.LocalDateTime
+
+import reactivemongo.bson.BSONObjectID
+
+case class Movie(image: String,
+                 title: String,
+                 director: String,
+                 description: String,
+                 actors: List[String],
+                 var screenings: List[DateTime])
+
+case class MovieWithID(
+                        _id: BSONObjectID,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name '_id' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)